extensions/cornerstone/src/services/ToolService/ToolService.ts

```diff
@@ -0,0 +1,152 @@
+import { Types as OhifTypes, ServicesManager, PubSubService } from '@ohif/core';
+import { eventTarget } from '@cornerstonejs/core';
+import { Enums as cstEnums } from '@cornerstonejs/tools';
+import { CornerstoneServices } from '../../types';
+
+const EVENTS = {
+  TOOL_ACTIVATED: 'event::cornerstone::toolservice:toolActivated',
+  PRIMARY_TOOL_ACTIVATED:
+    'event::cornerstone::toolservice:primaryToolActivated',
+};
+
+class ToolService extends PubSubService {
```

Comment:
I don't understand why we need another service. It appears that you are only subscribing to an event and just publishing again. You can do that in the init.ts for the cornerstone extension

Reply:
if you really need a service I recommend you using `toolGroupService` for this

Reply:
Updated